torch/lib/THD/base/data_channels/DataChannelGloo.cpp

```diff
@@ -40,7 +66,11 @@ DataChannelGloo::~DataChannelGloo() {
 
 
 bool DataChannelGloo::init() {
-  getFullMeshCtx(); // wait for everyone to connect
+  for (uint8_t i = 0; i < (uint8_t)data_operation::last; i++) {
```

Comment:
Is it wise to depend on casting `enum class` objects to uints? (It looks like there are no obvious solutions though: https://stackoverflow.com/questions/2102582/how-can-i-count-the-items-in-an-enum)